### ISSUE TITLE
Graphic glitch after entering a command

### ISSUE THREAD

**Issue by stpoa** [shockone/black-screen#885]
#### Description

After typing a command and hitting enter, weird glitch shows on the screen for a split of a second. Image showing the problem below.

![screen shot 2016-11-13 at 02 18 21](https://cloud.githubusercontent.com/assets/19593179/20243071/afb09e3c-a947-11e6-9093-1739c8fd5217.png)

**Comment by tmas** [comment]
If you're referring to the dark bar on the prompt, that's actually an animation that shows the command is running. It does look a bit silly for super quick commands, though... Maybe it could be delayed a bit or fade in so it's never just a quick flash?

**Comment by stpoa** [comment]
I agree delaying it would be a very good idea, for me it looks like a bug when i enter a simple command like cd,ls,pwd etc. Another thing is a fat cursor showing in the next line at the same time.

**Comment by muescha** [comment]
i suggest a delay with the start of the animation

maybe it would be ok that the animation not start at first second but after a delay - so the glitch is not at short commands and on long running commands it works fine if then a animation starts
